wiki/Help_Centre/Account_Restrictions/vi.md

```diff
@@ -44,7 +44,7 @@ Nếu việc giới hạn tài khoản của bạn là chính đáng, chúng tô
 
 ## Tôi có thể làm gì khi đang chờ để khiếu nại?
 
-- **Tiếp tục tận hưởng game!** Khi tài khoản của bạn bị giới hạn, và bạn đang trong thời gian chờ, bạn vẫn có thể tiếp tục chơ game (miễn là ở tài khoản chính!). Điểm bạn ghi được khi bị giới hạn cũng sẽ được ghi nhận khi bạn được gỡ giới hạn (trừ khi bạn bị rollback).
+- **Tiếp tục tận hưởng game!** Khi tài khoản của bạn bị giới hạn, và bạn đang trong thời gian chờ, bạn vẫn có thể tiếp tục chơi game (miễn là ở tài khoản chính!). Điểm bạn ghi được khi bị giới hạn cũng sẽ được ghi nhận khi bạn được gỡ giới hạn (trừ khi bạn bị rollback).
 - **Giữ thái độ tốt.** Chúng tôi hiểu rằng việc bị giới hạn có thể gây khó chịu, nhưng hãy coi như đây là một cơ hội cho bạn tạm nghỉ, rời xa cộng đồng. Hãy tránh tương tác với cộng đồng trong một vài ngày (bất kể qua Discord, mạng xã hội, hay bất cứ hình thức liên lạc nào khác) để hối cải về hành động bạn đã làm khiến cho tài khoản bị giới hạn.
 - **Đảm bảo bạn tuân thủ nội quy.** Giới hạn dài hạn là hình phạt cho việc phạm quy. Nếu bạn vẫn tái diễn khi bị giới hạn, thời gian chờ để xin khiếu nại sẽ đếm lại về ba tháng kể từ lần gần đây nhất bạn vi phạm. Nó bao gồm tiếp tục gian lận, truy cập tài khoản người khác, hoặc tạo tài khoản mới để né giới hạn.
```

Comment:
Should "đếm lại" be replaced with "được tính lại"? It sounds a bit more "natural" I think.